shesmu-server-ui/src/definitions.ts

```diff
@@ -689,12 +693,39 @@ function testFunction(func: FunctionDefinition): void {
   const parsers = func.parameters.map(
     (p) => parseDescriptor(p.type, valueParser)[0]
   );
-  const inputs = func.parameters.map(() => inputText());
+  const savedFormat = locallyStoredString("shesmu_try_it_format", "shesmu");
+  const inputs = func.parameters.map(() => inputTextArea("", true));
   const errors = func.parameters.map(() => pane("blank"));
   dialog((_close) => [
     "Test function ",
     func.name,
     br(),
+    "Argument Format: ",
+    dropdown(
+      (format, selected) => {
+        if (format == "json") {
+          return [
+            { type: "icon", icon: "braces" },
```

Comment:
Could these be icon + text? The icon-only dropdowns have proven confusing to users

Reply:
Done!